core/Dockerfile

```diff
@@ -26,5 +25,9 @@ COPY ./cat /app/cat
 COPY ./install_plugin_dependencies.py /app/install_plugin_dependencies.py
 RUN python3 install_plugin_dependencies.py
 
+### ENVIRONMENT VARIABLES ###
+ENV PYTHONUNBUFFERED=1
```

Comment:
Please move those two lines up, just afet the `FROM`

Reply:
Additional changes:
- please redo the PR branching off `develop` and towards `develop` (`main` is only for release)
- also update `docker-compose.yml` and take away the unnecessary env vars `PYTHONUNBUFFERED` and `WATCHFILES_FORCE_POLLING`